tsconfig.json

```diff
@@ -1,17 +1,25 @@
 {
-  "include": ["./src/**/*.ts"],
+  "include": [
+    "./src/**/*.ts",
+    "src/index.ts"
```

Comment:
Please I would like to see why you want to include `src/index.ts`.

Reply:
Added by accident. removing